tests/python/pants_test/backend/jvm/subsystems/test_jvm_platform.py

```diff
@@ -139,3 +139,41 @@ def test_synthetic_target_runtime_platform_lookup(self):
             'target-runtime-platform')
     assert (instance.get_runtime_platform_for_target(
       synth_just_platform_with_parent_same).name == 'default-platform')
+
+  def test_jvm_options(self):
+    init_subsystem(JvmPlatform, options={
+      'jvm-platform': {
+        'platforms': {
+          'platform-with-jvm-options': {'target': '8', 'jvm_options':['-Dsomething']},
+          'platform-without-jvm-options': {'target': '8'},
+          'platform-with-jvm-options-needing-shlex': {'target': '8',
```

Comment:
Could the shlex parsing be lifting into a separate test?

Reply:
Yes. I thought it'd be easier to follow together this way, but I can break it out.